src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -4336,7 +4334,7 @@ class C
     void M()
     {
         int i = C.y;
-        var t = (({|Conflict:(int)C.y|}, ({|Conflict:(int)C.y|}, _)) = (1, (C.y, 3)));
+        var t = (({|Conflict:i|}, ({|Conflict:i|}, _)) = (1, (i: C.y, 3)));
```

Comment:
this now matches the normal conflict behavior we have, which is to not touch that location.  there was some extremely weird code just for deconstruction to try to still insert the expression value, even though we were in a write-location, not a read-location.